tasks/burp_suite_pro.yml

```diff
@@ -56,6 +56,7 @@
           "Where should Burp Suite Professional be installed\\?": "{{ burp_suite_install_directory }}"
           "Create symlinks\\?": "y"
           "Select the folder where you would like Burp Suite Professional to create symlinks, then click Next": "{{ burp_suite_symlinks_directory }}"
+        timeout: 600
```

Comment:
Just for clarification the PR description states two minutes, but this appears to be ten minutes. If I am not misinterpreting this, which is the actual desired timeout?

Reply:
Ha, the commit log says five minutes which was the actual intent.  Caught me slippin'!

Corrected in commit ecacbc6.